pages/common/test.md

```diff
@@ -8,9 +8,9 @@
 
 `test "{{$MY_VAR}}" == "{{/bin/zsh}}"`
 
-- Test if a given variable is empty:
+- Test if a given variable is empty[z]/[n]on-empty:
 
-`test -z "{{$GIT_BRANCH}}"`
+`test {{-z|-n}} "{{$GIT_BRANCH}}"`
```

Comment:
```suggestion
`test -{{z|n}} "${{variable}}"`
```

Optional fix: reduce duplication: `-` can be moved out of the placeholder.
Optional fix: consistency violation: second placeholder can be updated to conform the rest of pages.